packages/db/src/scripts/seed.ts

```diff
@@ -58,14 +58,14 @@ const loaddb = async () => {
     await prisma.exercise.deleteMany()
     logger('delete', 'exercise')
 
-    await prisma.log.deleteMany()
-    logger('delete', 'log')
+    await prisma.workoutLog.deleteMany()
+    logger('delete', 'workoutLog')
 
     await prisma.workout.deleteMany()
     logger('delete', 'workout')
 
-    await prisma.log.deleteMany()
-    logger('delete', 'log')
+    // await prisma.log.deleteMany()
```

Comment:
Why was this commented out?

Reply:
If it is because log changed to workoutLog then can we just rename it?

Reply:
If it is no longer needed can we just delete it?

Reply:
Hey, sorry about that; there were 2 delete statements for log(workoutLog now) before, and I commented one out to see if there was a need for it when I ran it. It has been removed now.